core/server/master/src/main/java/alluxio/master/DefaultAlluxioMaster.java

```diff
@@ -130,7 +135,8 @@ protected DefaultAlluxioMaster() {
       }
       mTransportProvider = TransportProvider.Factory.create();
       mTServerSocket =
-          new TServerSocket(NetworkAddressUtils.getBindAddress(ServiceType.MASTER_RPC));
+          new TServerSocket(NetworkAddressUtils.getBindAddress(ServiceType.MASTER_RPC),
+                  connectionTimeout);
```

Comment:
Hello, we will be closed in ThriftClientPool optimized connection, at this stage there may be no consider the other other type of the servers, the subsequent will continue to optimize the function.